client/lib/openseadragon.js

```diff
@@ -1,21 +1,52 @@
 require('openseadragon');
 var $ = require('jquery');
 
-module.exports = function (imgUrl, ctx) {
-  if (!$('#openseadragon').length) return;
+module.exports = {
+  init: function (ctx, imgUrl, cb) {
+    imgUrl = imgUrl || $('.slick-active img')[0].src;
+    if (!$('#openseadragon').length) return;
 
-  ctx.viewer = OpenSeadragon({
-    id: 'openseadragon',
-    // element: $('.record-imgpanel__singleimg'),
-    prefixUrl: '/assets/img/openseadragon/',
-    showZoomControl: true,
-    // just using an example static image, will be swapped for some magic tiles
-    tileSources: {
-      type: 'image',
-      url: imgUrl
+    if (!ctx.viewer) {
+      $('.record-imgpanel__slickwrap').addClass('hidden');
+      $('.record-imgpanel__dragon').removeClass('hidden');
+
+      ctx.viewer = OpenSeadragon({
+        id: 'openseadragon',
+        prefixUrl: '/assets/img/openseadragon/',
+        showZoomControl: true,
+        tileSources: {
+          type: 'image',
+          url: imgUrl
+        },
+        zoomInButton: 'osd-zoomin',
+        zoomOutButton: 'osd-zoomout',
+        fullPageButton: 'osd-fullpage',
+        homeButton: 'osd-home',
+        toolbar: 'openseadragon-toolbar',
+        gestureSettingsMouse: {
+          scrollToZoom: false
+        }
+      });
```

Comment:
Does this actually work? - if we save the context and then navigate away and back to the page the DOM element(s) will be different, does `OpenSeadragon` deal with this....?